rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -443,6 +443,36 @@ message RegisterDappCanisters {
   repeated ic_base_types.pb.v1.PrincipalId canister_ids = 1;
 }
 
+message PreciseValue {
+  oneof precise_value {
```

Comment:
```suggestion
  oneof precise_value_inner {
```

Reply:
I need to merge this PR to enable a demo. It is safe because clients cannot use the new API yet, so we are free to change it for the time being. I'll address this in the follow-up PR.